data/reusables/reminders/slack-channel.md

```diff
@@ -1 +1,3 @@
 1. Under "Slack channel", type the name of the Slack channel where you'd like to receive notifications.
+   > [!TIP]
+   > If this Slack channel is private, you will need to invite the integration into the channel: `/invite @github`. Alternatively, you can ask users to run `/github signin` in the Slack channel. For more information, see [Getting started](https://github.com/integrations/slack?tab=readme-ov-file#getting-started) in the Slack integrations documentation.
```

Comment:
Let me correct it (to the best of my knowledge)

1. Using `/invite @github` to a private channel is mandatory for the integration to work.
2. Running `/github signin` is not an alternative to running `/invite @github`, in fact the integration works and send reminders without it. Running `/github signin` for each users will allow them to get tagged as part of the reminder.

The last part about tagging of users is the missing part that need to be somewhere in the docs. Please see my original version of the PR.

Reply:
When saying tagging I meant, mention. So the user gets a notification and as a result, aware that they need to review a PR.

Reply:
Thank you for your clear review comments. I've updated the tip.